instrumentation/resources/library/src/main/java/io/opentelemetry/instrumentation/resources/ProcessResource.java

```diff
@@ -69,10 +72,33 @@ private static Resource doBuildResource() {
 
       attributes.put(ResourceAttributes.PROCESS_EXECUTABLE_PATH, executablePath.toString());
 
+      List<String> commandArgs = new ArrayList<>();
+      commandArgs.add(executablePath.toString());
+      commandArgs.addAll(runtime.getInputArguments()); // Only includes VM arguments
+      // sun.java.command doesn't seem to be well document or official, but there
+      // doesn't seem to be any better way to get this consistently without more complexity.
+      String javaCommand = System.getProperty("sun.java.command");
+      if (javaCommand != null) {
+        String[] args = javaCommand.split(" ");
+        if (args.length > 0) {
+          // TODO: add handling for windows paths.
+          if (args[0].startsWith("/")) {
+            // We are dealing with a `java -jar /path/to/some.jar` situation and need to add
+            commandArgs.add("-jar");
```

Comment:
Should prepend `-jar ` to `javaCommand ` to have it in `process.command_line` too?